custom_components/dreo/__init__.py

```diff
@@ -49,47 +53,74 @@ async def async_setup_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> b
     _LOGGER.debug("Device dict is: %s", device_dict)
 
     manager.start_transport()
-    
+
     hass.data[DOMAIN] = {}
     hass.data[DOMAIN][DREO_MANAGER] = manager
 
     fans = hass.data[DOMAIN][DREO_FANS] = []
     heaters = hass.data[DOMAIN][DREO_HEATERS] = []
-    platforms = []
+    acs = hass.data[DOMAIN][DREO_ACS] = []
+    cookers = hass.data[DOMAIN][DREO_COOKERS] = []
+    platforms = set()
 
     if device_dict[DREO_FANS]:
         fans.extend(device_dict[DREO_FANS])
-        platforms.append(Platform.FAN)
-        platforms.append(Platform.SENSOR)
-        platforms.append(Platform.SWITCH)
-        platforms.append(Platform.NUMBER)
+        platforms.add(Platform.FAN)
+        platforms.add(Platform.SENSOR)
+        platforms.add(Platform.SWITCH)
+        platforms.add(Platform.NUMBER)
 
     if device_dict[DREO_HEATERS]:
         heaters.extend(device_dict[DREO_HEATERS])
-        platforms.append(Platform.CLIMATE)
-        platforms.append(Platform.SENSOR)
-        platforms.append(Platform.SWITCH)
-        platforms.append(Platform.NUMBER)
-
+        platforms.add(Platform.CLIMATE)
+        platforms.add(Platform.SENSOR)
+        platforms.add(Platform.SWITCH)
+        platforms.add(Platform.NUMBER)
+
+    if device_dict[DREO_ACS]:
```

Comment:
Can you please make this AIRCONDITIONERS instead of ACs?  For ease of readability?